packages/web/components/Loading/Wrapper.js

```diff
@@ -1,20 +1,34 @@
-import React from 'react';
+import React, { createElement } from 'react';
 import PropTypes from 'prop-types';
 import Spinner from './Spinner';
+import { HiddenWrapper } from './styles';
 
-const Wrapper = ({ loading, variant, children }) => {
-	return loading ? <Spinner variant={variant} /> : children;
+const Wrapper = ({ loading, variant, children, alwaysRenderChildren }) => {
+	const spinnerComponent = createElement(Spinner, { variant });
+
+	if (alwaysRenderChildren) {
```

Comment:
Now I can use loading without just render children when the loading is finished. That's how the difference between `v-show` and `v-if` in vue.js